src/inputs/BasicInputs/BasicRadioInput/BasicRadioInput.js

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Radio, RadioGroup, FormControl, FormControlLabel, TextField } from '@material-ui/core';
+import { BasicInputWrapper } from '../BasicInputWrapper';
+import PropTypes from 'prop-types';
+
+export const BasicRadioInput = (props) => {
+  const {
+    label,
+    tooltipText,
+    value,
+    textFieldProps,
+    changeRadioOption,
+    enumValues,
+    radioGroupRow,
```

Comment:
The prop name `radioGroupRow` is redundant with the component name `BasicRadioInput`, could we rename the prop to `row` instead ?